lib/processor/zephyr_native/cpu.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/*
+ * @file	cpu.h
+ * @brief	Zephyr native test environment CPU specific primitives
+ */
+
+#include <stdint.h>
+
+#ifndef __METAL_ZEPHYR_NATIVE_CPU__H__
+#define __METAL_ZEPHYR_NATIVE_CPU__H__
+
+void k_busy_wait(uint32_t usec_to_wait);
+
```

Comment:
Shouldn't this come from Zephyr includes?
Does that complicate the build?  When building for Zephyr we are almost always executing from the Zephyr master cmake.  Even if not the user can supply the needed extra include path.

Reply:
It does in the latest version